specs-go/config.go

```diff
@@ -11,7 +11,7 @@ type Spec struct {
 	// Process configures the container process.
 	Process *Process `json:"process,omitempty"`
 	// Root configures the container's root filesystem.
-	Root Root `json:"root"`
+	Root *Root `json:"root,omitempty"`
```

Comment:
This may fall under the no-pointers-if-required-on-any-platform policy that @mrunalp [was going to document][1].  If so, I can drop the pointer (and `omitempty`?), but it would be nice to get that policy documented.

[1]: https://github.com/opencontainers/runtime-spec/pull/759#issuecomment-301199323